near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -107,7 +107,7 @@ impl NonFungibleToken {
         let mut this = Self {
             owner_id,
             extra_storage_in_bytes_per_token: 0,
-            owner_by_id: TreeMap::new(owner_by_id_prefix),
+            owner_by_id: UnorderedMap::new(owner_by_id_prefix),
```

Comment:
We can't just change this because it is state-breaking and will break for anyone trying to update their NFT version.

If you need this change, you probably just want to fork this logic